x/slashing/keeper.go

```diff
@@ -49,6 +49,12 @@ func (k Keeper) handleDoubleSign(ctx sdk.Context, addr crypto.Address, infractio
 		panic(fmt.Sprintf("Validator consensus-address %v not found", consAddr))
 	}
 
+	// Get validator.
+	validator := k.validatorSet.ValidatorByConsAddr(ctx, consAddr)
+	if validator == nil || validator.GetStatus() == sdk.Unbonded {
```

Comment:
Hmm - if `MaxEvidenceAge` is equal to the unbonding period, shouldn't this case be impossible? Is this an off-by-one problem due to the next val set offset? If so can we add a comment about it?

Reply:
Simulation doesn't take this into account, and Tendermint might break this assumption at some point.